docker/autotests.Dockerfile

```diff
@@ -0,0 +1,17 @@
+# We need a mysqldump binary in this image.
+# The simplest way to get this - use a mysql image as base.
+ARG MYSQL_VERSION=5.7
+FROM mysql:${MYSQL_VERSION}-debian
+ARG GOVERSION=1.18
+WORKDIR /src
+ENV PATH="$PATH:/usr/local/go/bin"
+RUN apt-get update && \
+    apt-get install --yes git curl build-essential && \
+    update-ca-certificates
+RUN curl -L -o go.tar.gz https://go.dev/dl/go${GOVERSION}.linux-amd64.tar.gz && tar -C /usr/local -xzf go.tar.gz
+COPY docker/resources/waitfor.sh /
+COPY go.* ./
+RUN go mod download
+COPY ./ .
+
+CMD go test -race -v -timeout 2m ./... -gocheck.v
```

Comment:
I'm not sure about your environment. The first thought comes to my mind is using docker to start a MySQL at 3306 and running `go test` outside the docker. The good point is if I change some code, I don't need to build the docker image again and no need to prune unused docker local volumes periodically.